resources/Materials/TestSuite/_options.mtlx

```diff
@@ -87,11 +87,11 @@
     <!-- Transforms UVs of loaded geometry -->
     <input name="transformUVs" type="matrix44" value="1.0f, 0.0f, 0.0f, 0.0f, 0.0f, -1.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, -0.235f, 1.0f, 0.0f, 1.0f" />
 
-    <!-- External testing: extra comma separated list of library paths -->
-    <input name="externalLibraryPaths" type="string" value="" />
+    <!-- List of extra library paths for generator and render tests -->
+    <input name="extraLibraryPaths" type="string" value="" />
 
-    <!-- External testing: extra comma separated list of test root paths -->
-    <input name="externalTestPaths" type="string" value="" />
+    <!-- List of document paths for render tests -->
+    <input name="renderTestPaths" type="string" value="resources/Materials/Examples/StandardSurface,resources/Materials/Examples/UsdPreviewSurface,resources/Materials/TestSuite" />
```

Comment:
This is great to externalize this!
Just want to check here that this is now all the test paths to check instead of extra ones. Thus if this is empty, nothing will be considered to run.

Reply:
That's correct, @bernardkwok, and this now represents the full set of paths to be used for render testing (across all languages).